content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -286,8 +286,8 @@ tracker instead of the kubeadm or kubernetes issue trackers.
 Several external projects provide Kubernetes Pod networks using CNI, some of which also
 support [Network Policy](/docs/concepts/services-networking/networkpolicies/).
 
-See the list of available
-[networking and network policy add-ons](/docs/concepts/cluster-administration/addons/#networking-and-network-policy).
+See a list of add-ons that implement the
+[Kubernetes networking model](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).
```

Comment:
ideally we should remove this page:
/docs/concepts/cluster-administration/addons/#networking-and-network-policy

it enumerates a similar list as in /docs/concepts/cluster-administration/networking